src/http/api/account.ts

```diff
@@ -136,20 +139,51 @@ export function createGetAccountFollowsHandler(
 
             const account = await accountRepository.getByApId(new URL(apId));
 
-            accountFollows = await accountFollowsView.getFollowsByHandle(
-                handle,
+            const result = await accountFollowsView.getFollowsByApId(
```

Comment:
Should we be calling `getFollowsByAccount` if `account` is present?

Reply:
Or we move the local existence check into the `getFollowsByApId` method 🤔 But right now the knowledge is kinda split between the two